lib/src/components/NxDivider/__tests__/NxDivider.test.tsx

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import { NxDivider } from '../NxDivider';
+import { getShallowComponent } from '../../../__testutils__/enzymeUtils';
+
+describe('NxDivider', function() {
```

Comment:
Can be simplified and moved to the SimpleComponents tests

Reply:
> Can be simplified and moved to the SimpleComponents tests

Oh I see it's already there too.  This needs to be deleted as well then.

Reply:
> > Can be simplified and moved to the SimpleComponents tests
> 
> Oh I see it's already there too. This needs to be deleted as well then.

Check e1f79dfe5abbe6861ef08f7b224747b2b200e2dd